src/lighteval/commands/lighteval_cli.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+# MIT License
+
+# Copyright (c) 2024 Taratra D. RAHARISON
```

Comment:
And the HuggingFace team :)

Reply:
Okay, I'll switch to the HuggingFace team. I didn't do it from the start because I thought it would have been inappropriate for me to act as a member of the HuggingFace team.

Reply:
I meant it should be you + HF, since you have authorship of these files as you are adding them :)

Reply:
Sorry, I misunderstood it. It's clearer now. Okay, I will correct it. I hope you have a nice weekend.